tests/unit/dataactbroker/test_generation_helper.py

```diff
@@ -577,3 +577,45 @@ def gen_file_path_from_submission(submission, original_filename):
     local_filepath = CONFIG_BROKER['broker_files']
     nonlocal_filepath = '{}/'.format(submission)
     return '{}{}'.format(local_filepath if CONFIG_BROKER['local'] else nonlocal_filepath, original_filename)
+
+
+@pytest.mark.usefixtures("job_constants")
+def test_check_detached_a_file_generation(database):
```

Comment:
This appears to be testing `check_file_generation` and just happens to be using an A file. There's nothing specific about A files being checked, just generation in general.